assets/scss/_nav.scss

```diff
@@ -121,13 +121,13 @@
             font-variant: normal;
             text-rendering: auto;
             -webkit-font-smoothing: antialiased;
-            font-family: "Font Awesome 5 Free";
-            font-weight: 900; 
+            font-family: "Font Awesome 6 Free";
```

Comment:
We should define a variable for the FA font name and use it here and elsewhere. This can be done in this PR or a followup PR.

Reply:
Will follow up via:

- #1170